markdown/cmark/config.h.in

```diff
@@ -0,0 +1,76 @@
+#ifndef CMARK_CONFIG_H
+#define CMARK_CONFIG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#cmakedefine HAVE_STDBOOL_H
+
+#ifdef HAVE_STDBOOL_H
+  #include <stdbool.h>
+#elif !defined(__cplusplus)
+  typedef char bool;
+#endif
+
+#cmakedefine HAVE___BUILTIN_EXPECT
```

Comment:
You might want to add a check for `__builtin_expect()` that `AC_DEFINE`s `HAVE___BUILTIN_EXPECT`.  I'm not sure how you check for this, but an `AC_TRY_COMPILE` would do I guess.